test/object-store/sync/flx_sync.cpp

```diff
@@ -295,7 +295,6 @@ static auto make_error_handler()
 }
 
 // Re-enable these tests in RCORE-1264 when the server websocket disconnect issues are resolved.
-#if 0
```

Comment:
These tests are now passing with some of the more recent BAAS changes - re-enabling test.

Reply:
Can you also remove the comment above about re-enabling these and close RCORE-1264?